docs/zilevm/developer-introduction.md

```diff
@@ -11,26 +11,36 @@ description: ZILEVM for developers
 
 ## Developer Introduction
 
-### Remix IDE
+If you are new to Solidity and looking to learn the basics of the language, there are a few guides that can provide a good starting point. These can include:
+
+- The Solidity documentation
```

Comment:
Link?

Reply:
```suggestion
- [The Solidity documentation](https://docs.soliditylang.org)
```

Reply:
Ahh, I see it is not intended to have a link

Reply:
I don't think it matters, does it? I mean, a nice to have and I 'll commit your suggestion if you like, but should this block merging?

Reply:
No, that's the point of my third comment - the first two are irrelevant